dockers/Screenshotter/screenshotter.js

```diff
@@ -202,6 +240,8 @@ function imageDimensions(img) {
 //////////////////////////////////////////////////////////////////////
 // Take the screenshots
 
+var countdown = listOfCases.length;
```

Comment:
It's not done in series, or at least not from our point of view. The tasks all get scheduled in parallel, and it's the Selenium implementation which serializes things again. Anyway, something like `Q.all` would be quite useful here, but introducing a second promise library into the game feels like a bad idea. Although it was just when I wrote this code that I first found out that the Selenium library even had a `Deferred` object. I had completely missed the “Types” section of [the API reference](http://seleniumhq.github.io/selenium/docs/api/javascript/index.html).